### DISCUSSION TITLE
License Fubar

### DISCUSSION THREAD

**Discussion by RadmaxTX** [aclap-dev/video-downloadhelper#1537] [Help and Q&A]
### Your question

I went to the website and bought a premium license.   I was using the firefox plugin but it seems to have possibly sold me a Chrome license even though I did not see anywhere to select a version.  My setting dialogs are different than what is shown in support posts and cant find anywhere to install or test the license. 

First 10 digits are 34629311b3

If I cant get this for Firefox I would like to exercise my right to a refund

### Addon & CoApp version and details

I did not have the button to copy the info. the first 10 of my license is 34629311b3

**Comment by jcfrog** [comment]
Hi,
Yes, you purchased a Chrome premium, do you want me to refund your Chrome premium so that you can purchase Firefox premium if you want to?
Do you have V8 or V9 interface?
